### PR TITLE
Dp 1151

### PR DESCRIPTION
**Pull request by shriharirathod**

### CODE REVIEW

File: usr/tgtd.c [hunk 1]
```diff
@@ -899,6 +955,13 @@ int main(int argc, char **argv)
 	ep_handlers->ha_endpoints[1].ha_user_data = NULL;
 	ep_handlers->ha_count += 1;
 
+	ep_handlers->ha_endpoints[2].ha_http_method = POST;
```
Comment (sumit-k) [inline usr/tgtd.c]: use ha_count instead of hard core values.

File: usr/tgtd.c [hunk 2]
```diff
@@ -817,6 +819,61 @@ static int lun_create(const _ha_request *reqp,
...
+	}
+
+	json_t *sport = json_object_get(root, "StordPort");
+	if (!json_is_string(sport)) {
+		set_err_msg(resp, TGT_ERR_INVALID_STORD_PORT,
+			"StordPort is not string");
+		return HA_CALLBACK_CONTINUE;
+	}
+
+	int ret = str_to_int_range((char *)json_string_value(sport),
```
Comment (sumit-k) [inline usr/tgtd.c]: Comments .. adhering to existing coding policies but the port range can go upto 64k.

File: usr/tgtd.c [hunk 3]
```diff
@@ -817,6 +819,61 @@ static int lun_create(const _ha_request *reqp,
...
+			stord_port, 1, 32768);
+	if (ret) {
+		set_err_msg(resp, TGT_ERR_INVALID_STORD_PORT,
+			"StordPort out of range");
+		return HA_CALLBACK_CONTINUE;
+
+	}
+
+	HycStorInitialize(1, hyc_argv, (char *)json_string_value(sip),
```
Comment (sumit-k) [inline usr/tgtd.c]: may be we need to do more error handling here later at some point.

Comment (shriharirathod) [reply usr/tgtd.c]: Added todo

File: usr/tgtd.c (hunk 2 above)
Comment (shriharirathod) [reply usr/tgtd.c]: Done

File: usr/tgtd.c (hunk 1 above)
Comment (shriharirathod) [reply usr/tgtd.c]: Done

**Pull request merged**